src/ucc/api/ucc.h

```diff
@@ -1384,32 +1400,41 @@ enum ucc_coll_op_args_field {
  *
  *  @b Description
  *  @n @n
- *  @ref ucc_coll_op_args_t defines the parameters that can be used to customize
+ *  @ref ucc_coll_args_t defines the parameters that can be used to customize
  *  the collective operation. The "mask" bit array fields are defined by @ref
- *  ucc_coll_op_args_field. The bits in "mask" bit array is defined by @ref
- *  ucc_coll_op_args_field, which correspond to fields in structure @ref
- *  ucc_coll_op_args_t. The valid fields of the structure are specified by
- *  setting the corresponding bit to "1" in the bit-array "mask". When bits
- *  corresponding to the fields are not set, the fields are not defined.
+ *  ucc_coll_args_field. The bits in "mask" bit array is defined by @ref
```

Comment:
ucc_coll_args_field -> ucc_coll_args_field

Reply:
We have struct: ucc_coll_args_t, but for some reason field is called "ucc_coll_arg_field_t". I think proper fix to call the field "ucc_coll_args_field_t". @manjugv ?

Reply:
> We have struct: ucc_coll_args_t, but for some reason field is called "ucc_coll_arg_field_t". I think proper fix to call the field "ucc_coll_args_field_t". @manjugv ?

Agree.